modules/statics/src/avax-p.ts

```diff
@@ -0,0 +1,82 @@
+import { BaseCoin, CoinFeature, CoinKind, KeyCurve, UnderlyingAsset } from './base';
+import { AvalancheNetwork } from './networks';
+
+export interface UtxoConstructorOptions {
+  fullName: string;
+  name: string;
+  network: AvalancheNetwork;
+  features: CoinFeature[];
+  asset: UnderlyingAsset;
+  prefix?: string;
+  suffix?: string;
+  primaryKeyCurve: KeyCurve;
+}
+
+export class UtxoCoin extends BaseCoin {
+  public static readonly DEFAULT_FEATURES = [
+    CoinFeature.UNSPENT_MODEL,
+    CoinFeature.CHILD_PAYS_FOR_PARENT,
+    CoinFeature.CUSTODY,
+  ];
+
+  /**
+   * Additional fields for utxo coins
+   */
+  public readonly network: AvalancheNetwork;
+
+  constructor(options: UtxoConstructorOptions) {
+    super({
+      ...options,
+      kind: CoinKind.CRYPTO,
+      isToken: false,
+      decimalPlaces: 8,
```

Comment:
Avax c has 9 decimal, becouse it's half of avax c

Reply:
fixed